build.rs

```diff
@@ -1,8 +1,34 @@
-use std::env;
-
 fn main() {
     // Forward the profile to the main compilation
-    println!("cargo:rustc-env=PROFILE={}", env::var("PROFILE").unwrap());
+    println!("cargo:rustc-env=PROFILE={}", std::env::var("PROFILE").unwrap());
     // Don't rebuild even if nothing changed
     println!("cargo:rerun-if-changed=build.rs");
+    // forward git repo hashes we build at
+    println!("cargo:rustc-env=GIT_HASH={}", get_commit_hash());
+    println!("cargo:rustc-env=COMMIT_DATE={}", get_commit_date());
+}
+
+pub fn get_commit_hash() -> String {
```

Comment:
these two functions are clippy independent and could be moved to the tools util crate

Reply:
moved